app/helpers/sessions_helper.rb

```diff
@@ -3,15 +3,36 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   def current_user
-    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+    if user_id = session[:user_id]
```

Comment:
Trong trường hợp có phép gán trong biểu thức điều kiện if thì sử dụng dấu ()
```suggestion
    if (user_id = session[:user_id])
```